assets/js/pages/AdvisoryDetails/AdvisoryDetails.test.jsx

```diff
@@ -7,11 +7,15 @@ import { faker } from '@faker-js/faker';
 import { renderWithRouter as render } from '@lib/test-utils';
 import { advisoryErrataFactory, cveFactory } from '@lib/test-utils/factories';
 
-import AdvisoryDetails from './AdvisoryDetails';
+import AdvisoryDetails, { formatPackage } from './AdvisoryDetails';
 
 describe('AdvisoryDetails', () => {
   it('displays a message, when the CVE, packages or fixes section is empty', () => {
-    const errata = advisoryErrataFactory.build({ cves: [], fixes: {} });
+    const errata = advisoryErrataFactory.build({
+      cves: [],
+      fixes: {},
+      affected_packages: {},
```

Comment:
```suggestion
      affected_packages: [],
```
If you `.buildList()` the factory in tests and `.map()` this in the implementation you're expeting an array, hence better have an empty array here